perf/yb-connect.yaml

```diff
@@ -14,10 +14,9 @@ services:
       HOST_NAME: "0.0.0.0"
       ADVERTISED_HOST_NAME: ${DOCKER_IP}
       KAFKA_JMX_PORT: '9875'
-      KAFKA_OPTS: "-javaagent:/usr/local/bin/jmx_prometheus_javaagent-0.9.jar=9876:/etc/jmx-exporter/kafka-connect.yml"
+      KAFKA_OPTS: "-javaagent:/kafka/etc/jmx_prometheus_javaagent-0.9.jar=9876:/etc/jmx-exporter/kafka-connect.yml"
     volumes:
       - ./jmx-exporter/kafka-connect.yml:/etc/jmx-exporter/kafka-connect.yml
```

Comment:
Can this also be added to the Dockerfile? Then all volumes can be eliminated.

Reply:
Yes, done